src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,34 @@
+import classnames from 'classnames';
+import { Error } from '../../types/Error';
+
+type Props = {
+  errorText: Error;
+  handleErrorChange: (error: Error | null) => void
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  errorText,
+  handleErrorChange,
+}) => {
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={classnames(
+        'notification is-danger is-light has-text-weight-normal',
+        { hidden: !errorText },
+      )}
+    >
+      <>
```

Comment:
is fragment necessary here?

Reply:
it wasn't working before, but now it's working for some reason